src/proto/package/tag.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package package;
+
+option go_package="market.goravel.dev/proto/package";
+
+import "google/api/annotations.proto";
+import "base/base.proto";
+
+message Tag {
+  string id = 1;
+  string user_id = 2;
+  string name = 3;
+  string created_at = 4;
+  string updated_at = 5;
+  string deleted_at = 6;
+}
+
+message TagsQuery {
+  string package_id = 1;
+  string name = 2;
+}
+
+message GetTagsRequest {
+  base.Pagination pagination = 1;
+  TagsQuery query = 2;
+}
+
+message GetTagsResponse {
+  base.Status status = 1;
+  repeated Tag tags = 2;
+  int64 total = 3;
+}
+
+service TagService {
+  rpc GetTags(GetTagsRequest) returns (GetTagsResponse) {
+    option (google.api.http) = {
+      get: "/tags"
+    };
+  }
+}
```

Comment:
Sorry, I missed one thing just now, this file is ok, but TagService is unnecessary, `GetTags` can be moved to `PacageService`, and the API link can be `/packages/tags`. Why add the prefix `/packages`, because we want to add a unified prefix for each service, eg: UserService: `/users/***`, PackageService: `/packages/***`.

Reply:
hmm, make sense